apps/core/src/components/Inputs/SendTokenFormInput.tsx

```diff
@@ -61,8 +63,8 @@ export function SendTokenFormInput({
 
     // gasBudgetEstimation should change when the amount above changes
     useEffect(() => {
-        setFieldValue('gasBudgetEst', formattedGasBudgetEstimation, false);
-    }, [formattedGasBudgetEstimation, setFieldValue, values.amount]);
+        setFieldValue('gasBudgetEst', gasBudgetEstimation, false);
```

Comment:
Since the `formattedGasBudgetEstimation` value is not stored now, we need to format it before displaying it in the review view
![image](https://github.com/user-attachments/assets/e31ccb99-bd1b-4ac2-914f-ff68ef979d3b)